@xen-orchestra/backups-cli/commands/clean-vms.js

```diff
@@ -326,9 +36,9 @@ module.exports = async function main(args) {
     // `@xen-orchestra/fs`.
     const release = await lockfile.lock(vmDir)
```

Comment:
You can remove lock handling from this file, and move it, either as a stub or a comment into `RemoteAdapter`.